packages/plugin-core/src/commands/pods/AirtableExportPodCommand.ts

```diff
@@ -186,14 +186,15 @@ export class AirtableExportPodCommand extends BaseExportPodCommand<
         records: records.created,
         engine,
         logger,
+        podId: config.podId,
```

Comment:
Do we need to make a corresponding change on the CLI side to include the podId?

Reply:
updated files